rs/bitcoin/ckbtc/minter/src/lib.rs

```diff
@@ -53,9 +54,9 @@ pub const MAX_REQUESTS_PER_BATCH: usize = 100;
 
 /// The constants used to compute the minter's fee to cover its own cycle consumption.
 /// The values are set to cover the cycle cost on a 28-node subnet.
```

Comment:
Note that this line is misleading. The costs for calls to the Bitcoin canister do not depend on the size of the subnet where the calling canister resides. The same holds for requesting ECDSA signatures.

Reply:
> The costs for calls to the Bitcoin canister do not depend on the size of the subnet where the calling canister resides.

I agree but this is just a subset of the costs of the minter's cycle consumption, no? I understand this comment as being about the total minter's cycle consumption, which in that case is also influenced by the subnet size. Or how would you formulate it to be less confusing?

Reply:
Good idea, done.